### PR TITLE
[docs] Add Data Balance README

### PR DESCRIPTION
**Pull request by ms-kashyap**
## Description

Changes made:
- Added a `databalance-README.md` to the repo, which is copy of [the data balance docs from SynapseML](https://microsoft.github.io/SynapseML/docs/features/responsible_ai/Data%20Balance%20Analysis/) with some changes
- Added a link from the main README to the data balance README
- Add images, which will be updated as we make changes to the UX (#1512)

The data balance README will serve as the main source of information from the dashboard. For example, the "Learn more" link from each set of measures will link to this README:
MicrosoftTeams-image (3)

In the future, we can move this information to [the responsibleai website](https://responsibleaitoolbox.ai/).

## Checklist

- [ ] I have added screenshots above for all UI changes.
- [ ] I have added e2e tests for all UI changes.
- [ ] Documentation was updated if it was needed.

### CODE REVIEW

**Review by imatiach-msft — APPROVED** [review]

File: docs/databalance-README.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+                        'occupation', 'relationship', 'race', 'gender', 'native-country']
+
+# Since a task type of 'classification' and a non-empty list of categorical features are specified,
+# data balance measures will be automatically added
+rai_insights = RAIInsights(model, train_data, test_data_sample, target_feature,
+                           task_type='classification', categorical_features=categorical_features)
+
+rai_insights.compute()
+
+ResponsibleAIDashboard(rai_insights)
```
Comment (imatiach-msft) [inline docs/databalance-README.md:58]: does this need to mention the flight currently?  I think this feature is under flight, so you need to pass the flight here?

**Comment by imatiach-msft** [comment]
@ms-kashyap  FYI it looks like this is failing the linter, make sure to run
```
yarn lintfix
```
to automatically fix the issues found by the linter.

File: docs/databalance-README.md (hunk 1 above)
Comment (ms-kashyap) [reply docs/databalance-README.md:58]: You're right! Thanks for catching this

File: docs/databalance-README.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+- The task type is `classification`.
+- A non-empty list of categorical features is specified.
+
+**Note: Data balance measures are currently being flighted. This means that to visualize them, you must specify `feature_flights="dataBalanceExperience"` in the `ResponsibleAIDashboard` constructor.**
+
+Data balance measures are automatically added for the following examples:
+
+- [Model debugging of a census income prediction model (classification)](https://github.com/microsoft/responsible-ai-toolbox/tree/main/notebooks/responsibleaidashboard/responsibleaidashboard-census-classification-model-debugging.ipynb)
+
+For each example, do the following:
```
[the remark this answers is not in the document]
Comment (imatiach-msft) [reply docs/databalance-README.md:43]: looks good to me

**Pull request merged**